middle_end/flambda2/from_lambda/lambda_to_flambda.ml

```diff
@@ -488,9 +511,13 @@ let rec cps acc env ccenv (lam : L.lambda) (k : cps_continuation)
           ap_loc ap_inlined ap_probe ap_result_layout k k_exn)
   | Lfunction func ->
     let id = Ident.create_local (name_for_function func) in
+    let id_duid = Flambda_debug_uid.none in
+    (* CR sspies: Is there a better [Flambda_debug_uid.t] available here? *)
```

Comment:
Hmm we should discuss, but we maybe need a predefined uid for "arbitrary closure".  It would be nice to propagate the uid properly though - maybe it could be added to Lambda.lfunction somehow?

Reply:
(also see comment below about this)

Reply:
Yeah but in fact we can just use the runtime block tags to identify closures, so I'm not sure there is anything to do here.  If we have a proper uid though that's good, as it means we could show the function type in the debugger.